packages/system-upgrade-controller/generated-changes/exclude/questions.yml

```diff
@@ -1 +1 @@
-rancher_min_version: 2.4.0-rc1
```

Comment:
Did you change this file manually? If so can you move it to the 1st commit? 
Also, since this file is excluded from the generated chart, why did you update it?

Reply:
I didn't change this at all. The file was updated upstream, so the exclude file was here was updated also.